src/services/role/utils/permissions.js

```diff
@@ -27,16 +27,16 @@ const definePermissions = (env, role, ...permissions) => {
 	if (['opt-out', 'enabled'].includes(Configuration.get(env))) {
 		// set defaul permission
 		Role.update({
-			name: role,
-		},
-		{
+				name: role,
+			},
```

Comment:
Codacy found an issue: [Expected indentation of 2 tabs but found 3.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-server/pullRequest?prid=6051638)